rotkehlchen/tests/api/test_evm_transactions.py

```diff
@@ -89,3 +89,48 @@ def test_query_transactions(rotkehlchen_api_server: 'APIServer'):
         for entry in result['entries']:
             assert entry['entry']['timestamp'] <= last_ts
             last_ts = entry['entry']['timestamp']
+
+
+@pytest.mark.parametrize('optimism_accounts', [[
+    '0xb8553D9ee35dd23BB96fbd679E651B929821969B',
+]])
```

Comment:
This address has a lot of transactions in optimism and the tested output overlaps with `test_query_transactions`.
Since you are testing that only optimism tx are queried doesn't it make more sense to use an address with fewer transactions? you can use mine `yabir.eth` (https://optimistic.etherscan.io/address/0xc37b40abdb939635068d3c5f13e7faf686f03b65) or any other that you find better